### ISSUE TITLE
"FR" should be labelled as "Switch to French"

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#214]
On the Onboarding screens:

The "Fr" should be labelled as "Switch to French".
The "En" should be labelled as "Switch to English".

**Comment by ForumG-SC** [comment]
Done
